Week-2/week2_form/style.css

```diff
@@ -0,0 +1,170 @@
+
+*{
+    margin:0px;
+    padding:0;
+    box-sizing:border-box;
+    font-size:15px;
+    font-family: "Lato", sans-serif;
+
+  }
+.wrapper{
+    display:grid;
+    grid-template-columns: repeat(2,1fr);
+    grid-template-rows:200px 200px;
+    width: 800px;
+    margin-left:230px;
+    justify-content: space-evenly;
+    align-content: space-between;
+    grid-row-gap: 20px;
+    margin-top:100px;
+}
+
+  header{
+    margin-top: 80px;
+    margin-left: 434px;
+    font-size: 33px;
+  }
+  
+  .grid-item {
+   
+    padding:20px;
+    font-weight: 100;
+    color:gray
+
+  }
+
+  img {
+   
```

Comment:
Try to avoid giving styles for generic HTML nodes, It may be troublesome to scale